core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -23,6 +23,7 @@ import java.net.URL
 import java.security.PrivilegedExceptionAction
 import java.text.ParseException
 import java.util.UUID
+import java.util.Collections
```

Comment:
This should be placed before `import java.util.UUID`. Please run `./dev/scalastyle` before pushing to branch.